cms/static/js/spec/video/transcripts/videolist_spec.js

```diff
@@ -8,7 +8,9 @@ define(
     ],
 function ($, _, Utils, VideoList, MetadataModel, AbstractEditor, sinon) {
     'use strict';
-    describe('CMS.Views.Metadata.VideoList', function () {
+    // TODO: reenable these specs, using AjaxHelpers and the correct pattern
+    // to avoid flakiness. See TE-970 for details.
+    xdescribe('CMS.Views.Metadata.VideoList', function () {
```

Comment:
@andy-armstrong I agree that we should not leave them disabled permanently.

I also feel strongly that we should disable them immediately, as they have been causing a lot of pain and wasted time, and also mistrust of the build system, for over a year.

I'll put in a ticket and reference it in the comment. I'll also bring the issue around to product and sustaining to see if they can prioritize it to be worked on right away. 

Thanks for any legwork, that'd be great if you could put any findings on the new JIRA ticket.